blocks/tabs/tabs.css

```diff
@@ -0,0 +1,275 @@
+@keyframes in-up {
+  0% {
+    opacity: 0;
+    transform: translate3d(0, 5px, 0);
+  }
+
+  100% {
+      opacity: 1;
+      transform: translate3d(0, 0, 0);
+  }
+}
+
+.tabs {
+  -webkit-font-smoothing: antialiased;
+
+  .tab-component {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+    width: 100vw;
+
+    p:last-child {
+      margin-bottom: 0;
+      display: flex;
+      justify-content: center;
+
+      picture {
+        display: block;
+        position: relative;
+        width: 90vw;
+        aspect-ratio: 1/1;
+      }
+
+      img {
+        position: absolute;
+        bottom: 0;
+        width: 100%;
+        height: 100%;
+      }
+    }
+
+    .tab-nav {
+      position: relative;
+      list-style-type: none;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      width: 96vw;
+      font-family: var(--calcite-sans-family);
+      padding: 0;
+      margin: 0 auto;
+
+      calcite-icon {
+        color: var(--calcite-ui-inverse);
+      }
+
+      .arrow-button {
+        position: absolute;
+        border: none;
+        padding: 0;
+        opacity: .96;
+        margin: auto 0;
+      }
+
+      .arrow-button.left {
+        left: 0;
+      }
+
+      .arrow-button.right {
+        right: 0;
+      }
+
+      .arrow-button[aria-hidden='true'] {
+        display: none;
+      }
+
+      .tab-title {
+        display: none;
+        border-bottom: 1px solid var(--calcite-ui-border-2);
+        padding-bottom: 2px;
+
+        button {
+          background: none;
+          color: white;
+          font-size: var(--font-0);
+          font-weight: 400;
+          padding: 0.75rem;
+        }
+      }
+
+      .tab-title[aria-hidden='false'] {
```

Comment:
Can you instead make this behaviour default, and change it to display: none for aria-hidden='true'? This way it's uniform across the board.